include/logging/log_msg.h

```diff
@@ -225,6 +225,14 @@ static inline uint32_t log_msg_timestamp_get(struct log_msg *msg)
 	return msg->hdr.timestamp;
 }
 
+/** @brief Convert timestamp of the message to us.
+ *
+ * @param timestamp Message timestamp
+ *
+ * @return Timestamp value in us.
+ */
+uint64_t log_msg_timestamp_to_us(uint32_t timestamp);
```

Comment:
since implementation is in `log_output.c` it would be better to put it in `log_output.h`.

Reply:
Okay, I move that there. I've also renamed function to prefix with `log_output_`.